.github/workflows/tests.yml

```diff
@@ -7,27 +7,35 @@ on:
 jobs:
   tests:
     name: PHP ${{ matrix.php }}; Symfony ${{ matrix.symfony }}
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
 
     strategy:
       matrix:
-        php: ['7.4', '8.0', '8.1', '8.2', '8.3', '8.4']
-        symfony: ['4', '5', '6', '7']
+        php: ['7.4', '8.0', '8.1', '8.2', '8.3', '8.4', '8.5']
+        symfony: ['5', '6', '7', '8']
         exclude:
           - php: '7.4'
             symfony: '6'
           - php: '7.4'
             symfony: '7'
+          - php: '7.4'
+            symfony: '8'
```

Comment:
symfony 8 requires PHP 8.4 🤔 

same for symfony 7.0 that required PHP 8.2

Reply:
This is an exclude list, not include.